bundles/io/org.eclipse.smarthome.io.net.security/README.md

```diff
@@ -0,0 +1,27 @@
+# NetworkSecurityTls
+
+This service is responsible for providing SSLContexts that can be used all over ESH
+for server applications like webservlets, Mqtt servers or any extension that need to
+provide for instance a REST interface.
+
+The SSLContext is configured to use either the user provided java keystore file
+(`../etc/default.keystore`) or if none is provided to generate a self-signed certificate
```

Comment:
Why a relative path of the current working directory? Shouldn't you use a path returned by a member of ConfigConstants? I also don't identify the place this location is used.